packages/gatsby/src/utils/__tests__/node-manifest.ts

```diff
@@ -24,11 +24,12 @@ jest.mock(`fs-extra`, () => {
       if (process.env.DEBUG) {
         console.log({ manifestFilePath, finalManifest })
       }
-      return {
-        manifestFilePath,
-        finalManifest,
-        then: (resolve): any => resolve(),
-      }
+      return new Promise(resolve =>
+        resolve({
+          manifestFilePath,
+          finalManifest,
+        })
+      )
```

Comment:
this works too but fyi you can call `Promise.resolve(value)` to create an immediately resolving promise :)

```
return Promise.resolve({
  manifestFilePath, finalManifest
})
```